todo_app/data/trello_items.py

```diff
@@ -0,0 +1,95 @@
+import os
+from todo_app.data.classes.Item import Item
+from todo_app.data.helpers.trello.get_list_for_trello_card import get_list_for_trello_card
+from todo_app.data.helpers.trello.make_request import make_request
+
+def get_items():
+    """
+    Fetches all saved Trello cards using the Trello REST API
+    and transforms them into app Items.
+
+    Returns:
+        items: The list of saved Items.
+    """
+
+    board_id = os.getenv('TRELLO_BOARD_ID')
+
+    url = f'https://api.trello.com/1/boards/{board_id}/lists'
+
+    query = {
+        'cards': 'open',
+        'card_fields': 'id,name'
+    }
+
+    lists = make_request(http_method='GET', url=url, query=query).json()
+
+    items = []
+
+    for list in lists:
+        for trello_card in list['cards']:
+            item = Item.from_trello_card(trello_card, list)
+            items.append(item)
+
+    return items
+
+def add_item(title):
+    """
+    Adds a new Item to Trello (in the form of a Trello card)
+    and returns this new Item.
+
+    Args:
+        title: The title of the Item.
+
+    Returns:
+        item: The new Item.
+    """
+
+    to_do_column_list_id = os.getenv('TRELLO_TO_DO_COLUMN_LIST_ID')
+
+    url = 'https://api.trello.com/1/cards'
+
+    query = {
+        'idList': to_do_column_list_id,
+        'name': title
+    }
+
+    trello_card = make_request(http_method='POST', url=url, query=query).json()
+
+    list = get_list_for_trello_card(trello_card)
+
+    item = Item.from_trello_card(trello_card, list)
+
+    return item
```

Comment:
Here you've got some code that's making a bonus network call to get everything into the right structure, but as far as I can see you just always ignore the result anyway (e.g. in `app.py` you ignore the result of this function). Is this actually a useful step?

Reply:
Makes a lot of sense; I'll admit it's a pet peeve of mine that our `session_items` implements it that way, but clearly another trainer disagreed when they wrote it!